src/Jobs/Recruit.Vacancies.Jobs/VacancyStatus/VacancyStatusJob.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+
+namespace Esfa.Recruit.Vacancies.Jobs.ApprenticeshipProgrammes
+{
+    public class VacancyStatusJob
+    {
+        private readonly ILogger<VacancyStatusJob> _logger;
+        private LiveVacancyStatusInspector _updater;
+
+        public VacancyStatusJob(ILogger<VacancyStatusJob> logger, LiveVacancyStatusInspector updater)
+        {
+            _logger = logger;
+            _updater = updater;
+        }
+
+#if DEBUG
+        public async Task UpdateStandardsAndFrameworks([TimerTrigger(Schedules.Midnight, RunOnStartup = true)] TimerInfo timerInfo, TextWriter log)
```

Comment:
Liking the use of DEBUG to make testing easier.  Rather than rewriting the signature maybe put the true/false into a const and just use the #if to set the const. Less intrusive perhaps.

Reply:
👍